code/game/objects/items/devices/scanners/guide.dm

```diff
@@ -86,7 +86,7 @@
 	if(organ)
 		dat += "<b>Organ damage</b> - Give Peridaxon. Perform full body scan for targeted organ repair surgery.<br>"
 	if(bloodloss)
-		dat += "<b>Low blood volume</b> - Commence blood transfusion via IV drip or provide blood-restorative chemicals (i.e. Iron)."
+		dat += "<b>Low blood volume</b> - Commence blood transfusion via IV drip or provide blood-restorative chemicals (e.g.: Copper for zorren and skrell, iron for the rest)."
```

Comment:
I'd put it more like "(i.e. Iron, or, for species such as Zorren or Skrell, Copper)". Basically putting the more generic and widely applicable one first.

Reply:
I think the two unique exceptions should be mentioned first, actually!

Reply:
I intended for writing iron first. However, i could not find a way to make it flow naturally. 

There's 2 exceptions, so higlighting the 2 exceptions first and then going "the rest" feels more comprehensible.